.github/workflows/coverage.yml

```diff
@@ -0,0 +1,32 @@
+name: Code Coverage
```

Comment:
This new CI job only runs for `master` branch, but while testing I enabled running on my test branch. You can see that run here (to check/confirm that it works properly):

https://github.com/ember-cli/ember-cli/runs/509294687?check_suite_focus=true